package.json

```diff
@@ -3,7 +3,7 @@
   "name": "mapeo-desktop",
   "description": "Mapping and monitoring in remote environments.",
   "productName": "Mapeo",
-  "version": "5.0.4",
+  "version": "5.0.6",
```

Comment:
this PR also includes a merge commit from master.. nasty but there's lots of big changes so it was the best way without having rebase hell